src/app/visual/list-view-grid/list-view-grid-visual.component.ts

```diff
@@ -21,4 +22,8 @@ export class ListViewGridTestComponent {
     { id: '6', column1: 606, column2: 'Lemon', column3: 'Larry eats lemons' },
     { id: '7', column1: 707, column2: 'Strawberry', column3: 'Sally eats strawberries' }
   ]);
+
+  public onToggleRowHighlightClick() {
+    this.rowHighlightedId = '2';
```

Comment:
The method doesn't actually toggle the highlight. What about something like?

```suggestion
    this.rowHighlightedId = (this.rowHighlightedId) ? undefined : '2';
```

Reply:
good call. done.